main.go

```diff
@@ -137,27 +120,18 @@ func run(kubeConfig string) error {
 	if err != nil {
 		return fmt.Errorf("error building network controllers: %v", err)
 	}
-	pdCtl := factory.Devices().V1beta1().PCIDevice()
-	pdcCtl := factory.Devices().V1beta1().PCIDeviceClaim()
-	nodeCtl := corecontrollers.Core().V1().Node()
+	pdCtl := pciFactory.Devices().V1beta1().PCIDevice()
+	pdcCtl := pciFactory.Devices().V1beta1().PCIDeviceClaim()
+	nodeCtl := coreFactory.Core().V1().Node()
```

Comment:
This is better naming ✅

Reply:
yeah that is what was causing the confusion 😆